tests/components/bmw_connected_drive/test_sensor.py

```diff
@@ -34,17 +34,17 @@ async def test_entity_state_attrs(
     ("entity_id", "unit_system", "value", "unit_of_measurement"),
     [
         ("sensor.i3_rex_remaining_range_total", METRIC, "279", "km"),
-        ("sensor.i3_rex_remaining_range_total", IMPERIAL, "173.36", "mi"),
+        ("sensor.i3_rex_remaining_range_total", IMPERIAL, "173", "mi"),
```

Comment:
Why remove the rest? Might as well include it now and use suggested_display_precision in the entity descriptions

Reply:
Set the display precision, but now had to add the full length converted value, as HA will print the full float as string state value.